Form/Type/SimpleMoneyType.php

```diff
@@ -31,7 +31,7 @@ public function __construct(PairManagerInterface $pairManager, $decimals)
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('tbbc_amount', new TextType())
+            ->add('tbbc_amount', 'Symfony\Component\Form\Extension\Core\Type\TextType')
```

Comment:
@allan-simon correct, but this syntax is only supported since PHP 5.5, I aimed to support 5.3.9 (as a minimal version required by symfony 2.8).